DEPENDENCIES.rst

```diff
@@ -22,45 +22,43 @@ In order to facilitate future compatibility with multiple platforms,
 Cyclus is built using ``CMake``. A full list of the Cyclus package
 dependencies is shown below:
 
-====================   ==================
-Package                Minimum Version
-====================   ==================
-``CMake``                3.16.3
-``boost``                1.71.0
-``libxml2``              2.9.10+
-``libxml++``             2.40.1
-``python``               3.8.10
-``sqlite3``              3.31.1
-``HDF5``                 1.10.4+
-``Coin-Cbc``             2.10.3
-``Coin-Clp``             1.17.5
-``zlib``                 1.2.11
-====================   ==================
+====================      ==================
+Package                   Minimum Version
+====================      ==================
+``CMake``                   3.16.3
+``pkg-config``              0.29.1
+``boost``                   1.71.0
+``libxml2``                 2.9.10
+``libxml++``                2.40.1
+``python`` (dev version)    3.8.10
+``sqlite3``                 3.31.1
+``HDF5``                    1.10.4
+``LAPACK``                  3.9.0
+====================      ==================
 
 .. website_include_end
 
 
 
 And a few optional dependencies:
 
-====================   ==================
-Package                Minimum Version
-====================   ==================
-doxygen (for docs)     1.7.6.1
-tcmalloc (for speed)   any
-Cython                 0.29+
-Python (dev version)   3.8+
-Jinja2                 any
-NumPy                  1.9+
-Pandas                 any
-====================   ==================
+====================   ==================  =============================================  ==================        
+Package                Minimum Version     Purpose                                        Notes
+====================   ==================  =============================================  ==================  
+``Coin-Cbc``             2.10.3             Enables use of Branch-and-Cut solver          Cyclus must be built with ``--allow-milps`` flag
+``Coin-Clp``             1.17.5             Enables use of Linear Programming solver      Cyclus must be built with ``--allow-milps`` flag
+``doxygen``              1.7.6.1            Building documentation
+``TCMmalloc``            any                Improves performance                          Only used if Cython is not present
+``Cython``               0.29               Enables use of Python agents and input files  
```

Comment:
I don't think these get separated out properly when this renders. I would go back and check on that.

Reply:
I believe I fixed the table spacing so they both show up in the website now (tested locally but please double check).  I also removed the note about installing the Python development version as the Python C API is included with standard python installations now

Reply:
The issue I found with these lines is when GitHub renders the table, not the website. The Coin-clp and doxygen end up in the same line, and the TCMmalloc and Cython end up in the same line, as shown in the figure. My guess is because the last column is blank for the doxygen and cython lines.
![Screenshot (3)](https://github.com/cyclus/cyclus/assets/26714848/5e414646-ecb6-4f51-a9ff-d260f91c2706)

Reply:
Ah I see.  Just added `N/A` in the blank cells and it is rendering correctly, is that an acceptable solution?